webapp/src/app/shared/table/table.component.css

```diff
@@ -241,6 +241,10 @@
     height: 100% !important;
 }
 
+.search-bar-cursor{
```

Comment:
Put that in a global css file and make class name generic, as this property is not specific to search-bar